src/lib.rs

```diff
@@ -44,8 +57,27 @@ enum ProcessState {
 /// }
 /// ```
 ///
+/// # MCCP2
```

Comment:
Could you put this doc in the [crate root](https://doc.rust-lang.org/1.30.0/book/first-edition/documentation.html#crate-documentation)?